src/containers/App.js

```diff
@@ -10,30 +12,41 @@ import { changeStatusColor } from '../utilities';
 function App(props) {
   return (
     <div className="container">
-    <div className="row">
-      <div className="six columns">
-        <h1>Air Tony</h1>
-        <InputForm fetchLocation={props.fetchLocation} />
-        <p style={{ color: changeStatusColor(props.message[1]) }}>{props.message[0]}</p>
-      </div>
-      <div className="six columns">
-        <CurrentLocation location={props.locations[0]} />
+      <div className="row">
+        <div className="six columns">
+          <h1>Air Tony</h1>
+          <InputForm
+            fetchLocation={props.fetchLocation}
+            changeStatus={props.changeStatus}
+          />
+          <p style={{ color: changeStatusColor(props.message[1]) }}>{props.message[0]}</p>
+        </div>
+        <div className="six columns">
+          <CurrentLocation location={props.locations[0]} />
+        </div>
       </div>
-    </div>
-    <div className="row">
-      <div className="twelve columns">
-        <HistoryTable locations={props.locations} />
+      <div className="row">
+        <div className="twelve columns">
+          <HistoryTable locations={props.locations} />
+        </div>
       </div>
     </div>
-  </div>
   );
 }
 
+App.propTypes = {
+  fetchLocation: PropTypes.func.isRequired,
+  changeStatus: PropTypes.func.isRequired,
+  locations: PropTypes.arrayOf(PropTypes.object).isRequired,
+  message: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
 const AppContainer = connect(
   function mapStateToProps(state) {
```

Comment:
destructure everywhere! instead of `(state)` use `({ locations, message, inputText })` then below return `{ locations, message, inputText }`